java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture3/tasks/_1/Number1.java

```diff
@@ -0,0 +1,203 @@
+                    orchestra.removeMusician(scanner.nextInt());
+                }
+                case 4 -> orchestra.play();
+                case 5 -> exit = true;
+                default -> System.out.println("ERROR");
+            }
+        }
+    }
+
+    public Musicians chooseInstrument() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Musician name:");
+        Musicians musicians = new Musicians(scanner.next());
+        System.out.print("Name is:");
+        musicians.getName();
+        System.out.println("What is his/her instrument?:");
+        System.out.println("1. Piano\n2. Violin\n3. Guitar");
+
+        switch (scanner.nextInt()) {
+            case 1 -> {
+                Piano piano = new Piano();
+                System.out.print("Instrument is: " + piano.getName());
+                musicians.setInstrument(piano);
+            }
+            case 2 -> {
+                Violin violin = new Violin();
+                System.out.print("Instrument is: " +  violin.getName());
+                musicians.setInstrument(violin);
+            }
+            case 3 -> {
+                Guitar guitar = new Guitar();
+                System.out.print("Instrument is: " + guitar.getName());
+                musicians.setInstrument(guitar);
+            }
+            default -> throw new IllegalStateException("Unexpected value: " + scanner.nextInt());
+        }
+        return musicians;
+    }
+}
+
+record Orchestra(ArrayList<Musicians> list) {
+
+    public void play() {
+        for (Musicians musicians : list) {
+            System.out.println(musicians.getSoundOfInstrument());
+        }
+    }
+
+    public void addMusician(Musicians musicians) {
+        this.list.add(musicians);
+    }
+
+    public void removeMusician(int integer) {
+        var removedMusician = this.list.remove(integer);
+        System.out.println("Removed" + removedMusician);
+    }
+
+    public void amountOfMusicians() {
+        System.out.println(list.size());
+    }
+
+    @Override
+    public String toString() {
+        return "Orchestra{" + "\n" + list +
+                '}';
+    }
+}
+
+class Musicians {
+    private static int id = 0;
+    private Instruments instrument;
+    private final int incrementId = ++id;
+    private final String name;
+
+    @Override
+    public String toString() {
+        return "Musician: " + "[id" + " " + incrementId + ", name of a musician = " + name + ']' + "\n" +
+                '[' + instrument + ']' + "\n";
+    }
+
+    public Musicians(String name) {
+        this.name = name;
+    }
+
+    public void getName() {
+        System.out.println(name);
+    }
+
+    public void setInstrument(Instruments instrument) {
+        this.instrument = instrument;
+    }
+
+    public String getSoundOfInstrument() {
+      return  instrument.getSound();
+    }
+}
+
+interface Instruments {
+    String name = "";
```

Comment:
It's better to leave methods without implementation. Think about a real-world example: what kind of instrument might not have a name or sound? Will it be a useful to have such instrument?